examples/test_util/data/EclIOdata.hpp

```diff
@@ -0,0 +1,51 @@
+/*
+  Copyright 2019 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPM_ECLIO_DATA_HPP
+#define OPM_ECLIO_DATA_HPP
+
+
+namespace examples {
+
+ namespace test_util {
+
+  namespace data {
```

Comment:
That was a highly nested namespace - I would suggest that everything is in the namespace `Opm::ecl`?

Reply:
See example below on suggested new function  block_size_data

Reply:
changed to Opm::ecl as you suggested